lib/TH/THTensorApply.h

```diff
@@ -370,77 +501,84 @@
  * Tensor. But because we are guaranteed the subsequent data is contiguous in memory, we
  * can simply loop for sizeof(A) iterations and perform the operation, without having to
  * follow the order described by the strides of A.
+ *
```

Comment:
I think that 'outermost' should be called 'innermost' in line 485, and outer/inner switched in general. Usually the first index is referred to as outermost.